README.md

```diff
@@ -18,7 +18,7 @@ Follow these steps to set up the application locally:
 2. **Install Required Packages**
 
    ```bash
-   pip install -r requirements.txt
+   uv sync
```

Comment:
Dodatkowo, sekcja "Prerequisites" powinna teraz zawierać informację nie o wymaganej wersji pythona, tylko o wymaganym uv, może nawet z gotową komendą do instalacji:

```shell
curl -LsSf https://astral.sh/uv/install.sh | sh
```

A w "Steps to Set Up the Application" nie potrzebujesz już "Create and Activate Virtual Environment", bo uv sync to robi za Ciebie :smile: